WordPress/UITestsFoundation/Screens/Me/SupportScreen.swift

```diff
@@ -9,52 +9,57 @@ public class SupportScreen: ScreenObject {
         $0.buttons["close-button"]
     }
 
-    private let contactSupportButtonGetter: (XCUIApplication) -> XCUIElement = {
-        $0.cells["contact-support-button"]
-    }
-
-    private let contactEmailFieldGetter: (XCUIApplication) -> XCUIElement = {
-        $0.textFields["Email"]
+    private let visitForumsButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.cells["visit-wordpress-forums-button"]
     }
 
     private let okButtonGetter: (XCUIApplication) -> XCUIElement = {
         $0.buttons["OK"]
     }
 
     var closeButton: XCUIElement { closeButtonGetter(app) }
-    var contactSupportButton: XCUIElement { contactSupportButtonGetter(app) }
-    var contactEmailField: XCUIElement { contactEmailFieldGetter(app) }
+    var visitForumsButton: XCUIElement { visitForumsButtonGetter(app) }
     var okButton: XCUIElement { okButtonGetter(app) }
 
     public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
             expectedElementGetters: [
                 closeButtonGetter,
-                contactSupportButtonGetter,
+                visitForumsButtonGetter,
                 // swiftlint:disable opening_brace
-                { $0.cells["help-center-link-button"] },
-                { $0.cells["my-tickets-button"] },
-                { $0.cells["set-contact-email-button"] },
+                { $0.cells["visit-wordpress-forums-prompt"] },
                 { $0.cells["activity-logs-button"] }
                 // swiftlint:enable opening_brace
             ],
             app: app,
-            waitTimeout: 7
+            waitTimeout: 22
         )
     }
 
-    public func contactSupport() throws -> ContactUsScreen {
-        contactSupportButton.tap()
-        addContactInformationIfNeeded()
-        return try ContactUsScreen()
+    public func assertVisitForumButtonEnabled() -> SupportScreen {
+        XCTAssert(visitForumsButton.isEnabled)
+        return self
     }
 
-    private func addContactInformationIfNeeded() {
-        if contactEmailField.waitForExistence(timeout: 3) {
-            contactEmailField.tap()
-            contactEmailField.typeText("user@test.zzz")
-            okButton.tap()
+    public func visitForums() {
+        let safari = XCUIApplication(bundleIdentifier: "com.apple.mobilesafari")
+        visitForumsButton.tap()
+
+        // Select the Address bar when Safari opens
+        let addressBar = safari.textFields["Address"]
```

Comment:
This part [fails](https://buildkite.com/automattic/wordpress-ios/builds/12447#018656d2-c61b-4599-abe7-4c16f58f4153) for iPad on CI because the address bar is not a `textField` but a `button` on iPadOS:

![SCR-20230216-ouf](https://user-images.githubusercontent.com/73365754/219461465-758b278b-bdc6-4527-bd0e-e12f2dd841b1.png)

I'm on the fence if using a universal locator which would work for both, or using a conditional statement is a better approach here - both will do their job.

Reply:
@pachlava thanks I did not know this (that on iPad it's a Button)!  I'm currently updating to do something that will work for both.  Thanks for sharing 🙇